.dotnet/src/Custom/OpenAIClient.cs

```diff
@@ -53,10 +54,12 @@ namespace OpenAI;
 [CodeGenSuppress("GetInternalAssistantMessageClientClient")]
 [CodeGenSuppress("GetInternalAssistantRunClientClient")]
 [CodeGenSuppress("GetInternalAssistantThreadClientClient")]
+[CodeGenSuppress("GetInternalUploadsClientClient")]
 [CodeGenSuppress("GetLegacyCompletionClientClient")]
 [CodeGenSuppress("GetModelClientClient")]
 [CodeGenSuppress("GetModerationClientClient")]
 [CodeGenSuppress("GetVectorStoreClientClient")]
+[CodeGenSuppress("GetVectorStoreClientClient")]
```

Comment:
I think this is an accidental duplicate

Reply:
Weird -- wonder how that got there. Yes, you're right. Removed.